dll/shellext/zipfldr/zippidl.hpp

```diff
@@ -15,20 +16,20 @@ enum ZipPidlType
 #include <pshpack1.h>
 struct ZipPidlEntry
 {
-    WORD cb;
-    BYTE MagicType;
+    USHORT cb; // This must be a USHORT for SHITEMID compatibility
```

Comment:
Why exactly?
in `SHITEMID` it is `WORD cb`.

Reply:
Fixed as `WORD`.